pkg/action/executor/k8s_controller.go

```diff
@@ -198,6 +204,31 @@ func (c *parentController) update(updatedSpec *k8sControllerSpec) error {
 	return err
 }
 
+func (c *parentController) revert() error {
+	oldPodSpecUnstructured, found, err := unstructured.NestedFieldCopy(c.backupObj.Object, "spec", "template", "spec")
+	if err != nil || !found {
+		return err
+	}
+	currentObj, err := c.clients.dynNamespacedClient.Get(context.TODO(), c.obj.GetName(), metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+	currentPodSpecUnstructured, found, err := unstructured.NestedFieldCopy(currentObj.Object, "spec", "template", "spec")
+	if err != nil || !found {
+		return err
+	}
+	if reflect.DeepEqual(oldPodSpecUnstructured, currentPodSpecUnstructured) {
```

Comment:
Probably drop an info msg here (probably v4). This should not happen if we have reached revert, right?